### PR TITLE
Fix autocomplete and blank drop down on gis form

### LINKED ISSUE #282: GIS Expo contribution form has a blank dropdown
**ISSUE**
Under the "Thematic Category" label, there are two drop-downs.  The first has no content of is disabled.

**ACCEPTANCE**
- [ ] Remove the empty dropdown or figure out why it isn't being populated

### LINKED ISSUE #289: Autocomplete for places in GIS Expo view missing
In MIRA this uses Typeahead and queries a custom GeoNames JSON endpoint. 

We might want to use the built-in QA endpoint for this.

### PR DESCRIPTION
**Pull request by little9**
Closes #282 
Closes #289

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -30,3 +35,5 @@
 //= require tufts
 //= require app
 
+
```
Comment (no-reply) [inline app/assets/javascripts/application.js]: cleanup newlines?

**Comment by no-reply** [comment]
This looks okay to me, but it's also quite a big PR and I'm thinking we will be doing ourselves and Tufts a favor if we could add some details to the commit message. Especially: Is there commentary we could offer on the `handlebars.js` and `typeahead.js` dependencies? Is it necessary to include `handlebars.js` directly or can we use the version vendored in Hyrax instead?)

**Comment by little9** [comment]
@tom I forgot about Handlebars being used in Hyrax and removed it in favor of the Hyrax version.

@mark-dce seemed to think that this code was created after DCE initially created MIRA. 

Typeahead is being used with their own GeoNames endpoint, which doesn't seem to be part of the MIRA codebase. If Mike is at standup tomorrow I'll see if he knows more about this code.

**Comment by no-reply** [comment]
Looks good. I'll merge when it's :green_heart:.

**Comment by no-reply** [comment]
@little9 looks like there are some poltergeist issues.

**Comment by little9** [comment]
@no-reply 💚 🍏 🥗

**Pull request merged**